upsertGitHubTag/deployment/index.js

```diff
@@ -149,6 +149,16 @@ function processEvent(event, callback) {
   var loneEvent = event.Records[0];
   var requestBody = JSON.parse(loneEvent.body);
 
+  // Print SQS message ID if it is there
+  if (
+    Object.prototype.hasOwnProperty.call(loneEvent, "messageId") &&
```

Comment:
I would  just do `if (loneEvent.messageId)` -- looking at [the doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty), you're getting into details of whether the property is inherited or not, which I don't think matters, and depending on how the inheritance (or gets modified), may cause the test to fail. If it's there, log it.

And on further thought, perhaps just log it always, even if it is null or undefined? Wouldn't we want to know?

Reply:
Seconding always logging, might be nice to see if we're getting null IDs from anywhere.

Reply:
OK fixed please check my javascript